chatbot_ner/config.py

```diff
@@ -57,7 +57,12 @@
 ES_AUTH_PASSWORD = os.environ.get('ES_AUTH_PASSWORD')
 ES_BULK_MSG_SIZE = os.environ.get('ES_BULK_MSG_SIZE', '10000')
 ES_SEARCH_SIZE = os.environ.get('ES_SEARCH_SIZE', '10000')
-
+ES_INDEX_1 = os.environ.get('ES_INDEX_1')
```

Comment:
Can you please reorganise the order a little bit and put in inline and new comments for existing constants as well. For example - There are 3 index names, ES_INDEX_NAME, ES_INDEX_1, ES_INDEX_2. There are 2 URLS. So please keep in proper order/ grouping and put inline comments

Reply:
Done